source/_references/imagemetadata.md

```diff
@@ -1,85 +1,5 @@
 ---
 title: Image metadata
-use: [references]
```

Comment:
does this work to keep the page existing for search engine hits or bookmarks, but not show up in the index?

Reply:
this does not work, the page still shows up in the index. a quick search found nothing on how to hide pages in sculpin. @ChristianRiesen would you happen to know? otherwise we could install the sculpin redirect bundle and remove the page and configure a redirect...

Reply:
The only real way to hand this is a redirect. You can make pages that don'
t show up linked, because you just put them in a place where they are built but then you don't have an overview of any kind over them. But that wouldn't work here since we want to preserve the url.

Reply:
okay. then i will do a pull request to add the redirect tomorrow.